extensions/amp-analytics/0.1/events.js

```diff
@@ -177,20 +177,21 @@ class SignalTrackerDef {
 
 /**
  * The analytics event.
+ * @dict
  */
 export class AnalyticsEvent {
   /**
    * @param {!Element} target The most relevant target element.
    * @param {string} type The type of event.
-   * @param {!Object<string, string>=} opt_vars A map of vars and their values.
+   * @param {?JsonObject=} opt_vars A map of vars and their values.
    */
   constructor(target, type, opt_vars) {
     /** @const */
-    this.target = target;
+    this['target'] = target;
     /** @const */
-    this.type = type;
+    this['type'] = type;
     /** @const */
-    this.vars = opt_vars || Object.create(null);
+    this['vars'] = opt_vars || dict();
```

Comment:
hmm I wonder why we have this class at the first place, why not use a AnalyticsEventDef?  cc @lannka ? Ideas?